### PR TITLE
Issue #1163 - Convert ConvertLatLon to use template

### PR DESCRIPTION
**Pull request by Mailkov**
-----
IA Page: https://duck.co/ia/view/convert_lat_lon

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata post coming soon

**Comment by mintsoft** [comment]
@Mailkov Looks good, the only thing that bothers me is the subtitle:
image

`Convert in <unit>` doesn't quite read right to me, `Convert into <unit>` would work, or `Convert to <unit>` sounds better imo.

**Comment by moollaza** [comment]
Deploying to beta...

File: lib/DDG/Goodie/ConvertLatLon.pm [hunk 1]
```diff
@@ -213,9 +214,14 @@ handle query_nowhitespace => sub {
...
-    my $html = wrap_html(\@queries, \@results, $toFormat);
-
-    return $answer, html => $html;
+    my $result = join(", ",@results);
+
+    return $answer,
+    structured_answer => {
+        input     => [@queries],
+        operation => "Convert to $toFormat",
+        result    => html_enc($result),
```
Comment (mintsoft) [inline lib/DDG/Goodie/ConvertLatLon.pm]: I don't think this needs `html_enc` as long as the result is returned in unicode

**Comment by moollaza** [comment]
@Mailkov looks like it's triggering on queries with broken numbers, by ignoring the broken parts -- I think we should just ignore the whole query?

e.g. https://beta.duckduckgo.com/?q=71+degrees+10+minutes+3+foo+east+in+decimal&ia=answer

**Comment by Mailkov** [comment]
@moollaza I think is now ok

**Comment by moollaza** [comment]
Thanks! This LGTM :+1:

File: lib/DDG/Goodie/ConvertLatLon.pm (hunk 1 above)
Comment (moollaza) [inline lib/DDG/Goodie/ConvertLatLon.pm]: :+1:  I don't think there's risk of any XSS here, we can ditch the `html_enc` if we'd like

**Pull request merged**